File: openrest4j-api/src/com/googlecode/openrest/Notifications.java
package com.googlecode.openrest;

import java.io.Serializable;
import java.util.Collections;
import java.util.List;

import org.codehaus.jackson.annotate.JsonIgnoreProperties;
import org.codehaus.jackson.map.annotate.JsonSerialize;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Notifications implements Serializable {
    public Notifications(List<Notification> notifications) {
        this.notifications = notifications;
    }

    /** Default constructor for JSON deserialization. */
    public Notifications() {}

    @JsonSerialize(include = JsonSerialize.Inclusion.NON_DEFAULT)
    public List<Notification> notifications = Collections.emptyList();

    private static final long serialVersionUID = 1L;
}

File: trunk/openrest4net/openrest4net-api/com/googlecode/openrest/Notifications.cs
using System;
using System.Collections.Generic;

namespace com.googlecode.openrest
{
    public class Notifications
    {
        public Notifications(IList<Notification> notifications)
        {
            this.notifications = notifications;
        }

        /** Empty constructor required for initialization from JSON-encoded string. */
        public Notifications() { }

        public IList<Notification> notifications = new List<Notification>();
    }
}
